projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-users-endpoint.php

```diff
@@ -231,6 +231,21 @@ function ( $viewer ) use ( $args ) {
 
 					// When viewers are included, we ignore the order & orderby parameters.
 					if ( $include_viewers ) {
+						$viewer_ids     = array_map(
+							function ( $viewer ) {
+								return $viewer->ID;
+							},
+							$viewers
+						);
+						$combined_users = array_map(
```

Comment:
This API does not dedupe users or the count where there are users with WordPress roles that were once viewers: they'll be included in the viewers list and merged as is. 

See: https://github.com/Automattic/jetpack/pull/28317#issuecomment-2658129481